src/main/java/com/minecolonies/coremod/blocks/IBlockMinecolonies.java

```diff
@@ -0,0 +1,21 @@
+package com.minecolonies.coremod.blocks;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraftforge.fml.common.registry.IForgeRegistry;
+
+public interface IBlockMinecolonies
+{
+    /**
+     * Registery block at gameregistry.
+     * @param registry the registry to use.
+     * @return the block itself.
+     */
+    public IBlockMinecolonies registerBlock(final IForgeRegistry<Block> registry);
+
+    /**
+     * Registery block at gameregistry.
+     * @param registry the registry to use.
+     */
+    public void registerItemBlock(final IForgeRegistry<Item> registry);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "public" is redundant in this context. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2333)